src/BudgetManager/BudgetManager.API/Controllers/V1/TransactionController.cs

```diff
@@ -16,6 +16,15 @@ public class TransactionController : ControllerBase
         private readonly IMediator _mediator;
         public TransactionController(IMediator mediator) => _mediator = mediator;
 
+        [HttpGet("Recents")]
+        public async Task<IActionResult> GetTopRecents(CancellationToken cancellationToken)
+        {
+            var userId = Guid.Parse(User.FindFirst("UserId").Value);
+            var result = await _mediator.Send(new GetTop10RecentTransactionsQuery(userId), cancellationToken);
+
```

Comment:
I don't think the amount of transactions should be hard-coded, better pass a value.

Reply:
I don't know but in user stories, it asks to display ten recent transaction records.

Reply:
It is just a better practice to not hard-code it in the name, at least pass 10 by default in that case